src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java

```diff
@@ -73,7 +73,7 @@ public RetryingCommandHandler(final JedisPool connectionPool, final int maxRetri
 
     if (retries > 0) {
       log.warn("Redis communication error recovered after {}ms and {} of {} retries for command {} on {}",
-          currentTimeMillis() - methodInvocationStart, retries, maxRetries, command.getClass().getName(),
+          currentTimeMillis() - methodInvocationStart, retries + 1, maxRetries, command.getClass().getName(),
```

Comment:
Why not just call System.currentTimeMillis() here?

Reply:
See above. Don't mind changing it though